docs/toc.js

```diff
@@ -101,6 +101,10 @@ module.exports = {
           ["Get Performance Data", "get-performance-data.md"],
           ["Performance Data Types", "performance-data-types.md"],
         ]],
+        ["Recording Screen", ["recording-screen",
```

Comment:
I would expect this to be phrased as either `Screen Recording` or `Recording the Screen`

Reply:
Updated.